homeassistant/components/onboarding/views.py

```diff
@@ -100,14 +101,33 @@ class UserOnboardingView(_BaseOnboardingView):
                     data['name'], user_id=user.id
                 )
 
+            # Create default areas using the users supplied language.
+            translations = \
+                await hass.helpers.translation.async_get_translations(
+                    data['language'])
+
+            area_registry = \
+                await hass.helpers.area_registry.async_get_registry()
+
+            for area in ('living_room', 'bedroom', 'kitchen'):
```

Comment:
Do we expect this tuple to grow in the future? Then it's better to break out a constant to keep edits clean.

Reply:
Done